### PR TITLE
[TT-1426] Pings Proper Group for Compatibility Test Failures

### PR DESCRIPTION
**Pull request by kalverra**

### CODE REVIEW

File: .github/workflows/ccip-client-compatibility-tests.yml [hunk 1]
```diff
@@ -290,7 +290,6 @@ jobs:
           tag_suffix: ""
           dockerfile: core/chainlink.Dockerfile
           git_commit_sha: ${{ needs.select-versions.outputs.chainlink_version }}
-          check_image_exists: 'true'
```
Comment (Tofel) [inline .github/workflows/ccip-client-compatibility-tests.yml]: why?

File: .github/workflows/ccip-client-compatibility-tests.yml [hunk 2]
```diff
@@ -613,7 +612,7 @@ jobs:
                       "type": "section",
                       "text": {
                         "type": "mrkdwn",
-                        "text": "${{ contains(join(needs.*.result, ','), 'failure') && 'Some tests failed, notifying <@U03GWA0NSF7>' || 'All Good!' }}"
+                        "text": "${{ contains(join(needs.*.result, ','), 'failure') && 'Some tests failed! Notifying <!subteam^S06HJ46A1EX>' || 'All Good!' }}"
```
Comment (Tofel) [inline .github/workflows/ccip-client-compatibility-tests.yml]: nice one!

File: .github/workflows/ccip-client-compatibility-tests.yml (hunk 1 above)
Comment (kalverra) [reply .github/workflows/ccip-client-compatibility-tests.yml]: [That input is getting deprecation warnings](https://github.com/smartcontractkit/ccip/actions/runs/10156732455).

**Review by gheorghestrimtu — APPROVED** [review]

File: .github/workflows/ccip-client-compatibility-tests.yml (hunk 1 above)
Comment (Tofel) [reply .github/workflows/ccip-client-compatibility-tests.yml]: that's probably because this action wasn't updated and is behind chainlink

**Review by b-gopalswami — APPROVED** [review]

**Review by AnieeG — APPROVED** [review]

**Pull request merged**